scripts/plot-polarizer.py

```diff
@@ -78,14 +83,18 @@ def main(args):
         if "." not in options.filename[-5:]:
             raise ValueError("Output argument must contain extension, "
                              "but got '%s'" % (options.filename,))
+
+        amax = math.pi * 1.25  # 180°
         if options.polarizer == "linear":
             role = "lin-pol"
-            amax = math.pi  # 180°
         else: # qwp
             role = "quarter-wave-plate"
-            amax = math.pi / 2  # 90°
+            logging.info("Moving the linear polarizer to horizontal")
+            linpol = model.getComponent(role="lin-pol")
+            linpol.moveAbsSync({"rz": math.radians(45)})  # for RHC
```

Comment:
Add a comment why you move linpol to 45 degree when testing the qwp. What position do you need for qwp when testing the linpol or is this tested without the qwp?